feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -70,6 +130,19 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
       })
     );
   }
+  protected customerTicketingAssociatedObjectsPreConditions(): Observable<string> {
+    return this.userIdService.getUserId().pipe(
+      take(1),
+      map((userId) => {
+        if (!userId) {
+          throw new Error(
+            'Customer ticketing associated objects pre conditions not met'
```

Comment:
Is this a message that can be seen by the customer?  If so, you need to make it transalatable

Reply:
You can re-use the `customerTicketingPreConditions` as it returns a tuple.

The current logic says if ticketId doesn't exist, throw an error. We can just remove that condition and make it able to return a tuple of [userId, ticketId || undefined]. The way to do this is to make sure your second stream that is being combined can startWith `undefined`

Reply:
Because the way I see it, we don't always need the ticketId. Only for certain cases when it's within the ticket details page.